src/server/lang.js

```diff
@@ -0,0 +1,43 @@
+import path from "path"
+import {readdir} from "fs"
+
+import {addLocaleData} from "react-intl"
+import fr from "react-intl/locale-data/fr"
+import en from "react-intl/locale-data/en"
+
+const DEFAULT_LANGUAGE = 'fr'
+
+export function loadTranslations(langDir){
+  addLocaleData(...fr)
+  addLocaleData(...en)
+
+  var messages = new Map()
+  var dotIndex = -1
+  var json
+  readdir(langDir, (err, files) => { 
+    if(err){
+      console.log("Unable to load translation files.", err)
+    }
+
+    files.forEach(file => {
+      dotIndex = file.indexOf('.')
+      json = path.resolve(langDir, file)
```

Comment:
This is not actually a `json`, this is the path to a set of locale translations.